### PR TITLE
Rebuild images with Debian 12 + Add new suites

### LINKED ISSUE #8: [Feature request] Upgrade the python version on images
Hi,

It would be interesting according to me to be able to select a newer version of python on some images, per example python 3.8 or higher, to be able to make tests with some protocol implementations (aioquic for example which requires at least python 3.8).

Thanks in advance

### PR DESCRIPTION
**Pull request by Skazza94**
**Description**:
This PR updates the Kathará official Docker images. The primary focus of this update is to rebuild the images using Debian 12, a new and up-to-date base, and to introduce additional suites. The key contributions in this PR are outlined below:

1. **Upgrade to Debian 12:**
- The Docker images have been rebuilt using Debian 12 as the base distribution. This transition ensures that the Kathará framework utilizes the latest system improvements, security enhancements, and software packages provided by Debian 12.

2. **Introduction of New Suites:**
   Alongside the Debian 12 upgrade, this PR brings in a set of new suites that offer diverse environments for users to work with. These suites extend the capabilities of the framework, catering to a wider range of use cases and preferences. The complete list includes:
- `kathara/base`: used to build all other images. It contains a variety of network tools and some complex services like bind, apache, etc.
- `kathara/quagga`: extends the base image adding [Quagga](https://www.nongnu.org/quagga/).
- `kathara/frr`: extends the base image adding [FRRouting](https://frrouting.org/).
- `kathara/openbgpd`: extend the base image adding the [OpenBGPD daemon](https://www.openbgpd.org/).
- `kathara/krill`: extends the base image adding [Krill RPKI Certificate Authority](https://www.nlnetlabs.nl/projects/rpki/krill/).
- `kathara/routinator`: extends the base image adding [Routinator RPKI Relying Party](https://www.nlnetlabs.nl/projects/rpki/routinator/).
- `kathara/rpki-client`: extends the base image adding [OpenBGPD RPKI Client](https://www.rpki-client.org).
- `kathara/bird`: extends the base image adding [BIRD](https://bird.network.cz/).
- `kathara/rift-python`: extends the base image adding [Routing In Fat Trees (RIFT) Python Implementation](https://github.com/brunorijsman/rift-python).
- `kathara/sdn`: extends the base image adding [OpenVSwitch](https://www.openvswitch.org/) and [Ryu SDN controller](https://osrg.github.io/ryu/).
- `kathara/p4`: extends the base image adding [Behavioral Model (bmv2)](https://github.com/p4lang/behavioral-model) to compile and run P4-compliant programmable switches.

### CODE REVIEW

**Review by tcaiazzi — APPROVED** [review]

File: base/Dockerfile
Comment (lorenzo93) [inline base/Dockerfile:1]: Missing `mtr` package according to [this](https://github.com/KatharaFramework/Kathara/issues/238) issue in Kathará

**Review by lorenzo93 — CHANGES_REQUESTED** [review]
Everything ok except one single comment. :)

File: base/Dockerfile
Comment (Skazza94) [reply base/Dockerfile:1]: Hi @lorenzo93, added!

Comment (lorenzo93) [reply base/Dockerfile:1]: Dear @Skazza94 ,

`mrt` stands for "Multi-Threaded Routing Toolkit" and has nothing to do with `mtr`that stands for "My-Traceroute". I think you mispelled the package name, in fact, the `mrt` package does not even exists in the debian repositories. :)

**Review by lorenzo93 — APPROVED** [review]

**Pull request merged**